### ISSUE TITLE
README.md needs more info about iso-rename.sh

### ISSUE THREAD

**Issue by s243a** [techrockedge/archpup32#10]
The read-me needs more instructions about the iso-rename.sh script. I ran the script. It seems to have renamed my iso, from:
`Arch32Pup-20.03.iso`
to
`Arch32Pup-20.03+0.iso`

and has also created the delta:
Arch32Pup-20.03+0.iso___Arch32Pup-20.03+0.iso.delta 

but all the delta does is converts it to the same iso! Perhanps there should be a folder for the original Arch32Pup-20.03.iso, so that the xdelta program can compare to the orginal iso. Presumably, I'm supposed to modify the script each run to change some of the following variables:
```
version=20.03
baseversion=20.03+0

newversion=20.03+0

branch1=
branch2=
```

**Comment by peabee** [comment]
> Presumably, I'm supposed to modify the script each run to change some of the following variables:

Yes - iso-rename was not really meant for public use - just my little script to keep track on delta updates..... the first one can always be thrown away as +0 to +0 is trivial - more use for +1 etc.....
